app/assets/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/annotations.e2e.js.md

```diff
@@ -557,7 +557,7 @@ Generated by [AVA](https://ava.li).
         'skeleton',
       ],
       task: null,
-      tracingTime: 0,
+      tracingTime: null,
```

Comment:
Investigate tracingTime: null

Reply:
This is indeed weird. @fm3 Do you see a potential reason in this PR why the tracingTime in this snapshot would change?

Reply:
possibly unrelated to this PR, there are race conditions in time logging. https://github.com/scalableminds/webknossos/issues/2488

Reply:
hm, I wonder why this hasn't led to flaky builds in the past :thinking: anyway, the change would be fine by me for now if it doesn't increase "flakiness" for future builds.

Reply:
rebuild failed, so I’d suggest adding the 0 again in the snapshot. I believe the null case is fairly improbable. nonetheless, we should work on fixing those race conditions of course